OPHD/UI/StringTable.h

```diff
@@ -0,0 +1,86 @@
+#pragma once
+
+#include <NAS2D/Renderer/Renderer.h>
+#include <NAS2D/Renderer/Color.h>
+#include <NAS2D/Renderer/Point.h>
+#include <NAS2D/Renderer/Vector.h>
+#include <string>
+#include <vector>
+#include <cstddef>
+
+// Draw a 2 dimensional table of text. Determine cell size based on inserted text, font, and padding. Only allows one line of text per cell.
+class StringTable
+{
+public:
+	using CellCoordinate = NAS2D::Point<std::size_t>;
+
+	enum class Justification
+	{
+		Left,
+		Right
+	};
+
+	struct Cell
+	{
+		// Set textColor to ColorEmpty to indicate cell should use default StringTable color
+		static const NAS2D::Color ColorEmpty;
```

Comment:
I'm ok with it in either location. I debated both location without a strong preference. It ended up inside Cell since it typically shouldn't be applied to the Table's DefaultTextColor (although nothing would stop you from applying it as such I suppose.